cli/QuickstepCli.cpp

```diff
@@ -262,14 +262,19 @@ int main(int argc, char* argv[]) {
       DefaultsConfigurator::GetNumNUMANodesCoveredByWorkers(worker_cpu_affinities);
 
   if (quickstep::FLAGS_preload_buffer_pool) {
+    std::chrono::time_point<std::chrono::steady_clock> preLoadStart, preLoadEnd;
```

Comment:
For variable names, we use the `preload_start` style.

https://google.github.io/styleguide/cppguide.html#Variable_Names